tests/test_files/sample_test_file.md

```diff
@@ -5,27 +5,15 @@ The following is a list of test urls to extract.
  - [test url 3](https://github.com/SuperKogito/URLs-checker)
  - [test url 4](https://github.com/SuperKogito/URLs-checker/blob/master/README.md)
  - [test url 5](https://github.com/SuperKogito/URLs-checker/issues)
- - [test url 6](https://github.com/SuperKogito/URLs-checker/issues/1)
- - [test url 7](https://github.com/SuperKogito/URLs-checker/issues/2)
- - [test url 8](https://github.com/SuperKogito/URLs-checker/issues/3)
- - [test url 9](https://github.com/SuperKogito/URLs-checker/issues/4)
+ - [test url 6](https://github.com/SuperKogito/URLs-checker/issues/4)
 
-- [test url 10](https://github.com/SuperKogito/spafe/)
-- [test url 11](https://github.com/SuperKogito/spafe/issues)
-- [test url 12](https://github.com/SuperKogito/spafe/issues/1)
-- [test url 13](https://github.com/SuperKogito/spafe/issues/2)
-- [test url 14](https://github.com/SuperKogito/spafe/issues/3)
-- [test url 15](https://github.com/SuperKogito/spafe/issues/4)
-- [test url 16](https://github.com/SuperKogito/spafe/issues/5)
-- [test url 17](https://github.com/SuperKogito/spafe/issues/6)
-- [test url 18](https://github.com/SuperKogito/spafe/issues/7)
-- [test url 19](https://github.com/SuperKogito/spafe/issues/8)
+- [test url 7](https://github.com/SuperKogito/spafe/)
+- [test url 8](https://github.com/SuperKogito/spafe/issues)
+- [test url 9](https://github.com/SuperKogito/spafe/issues/1)
 
-- [test url 20](https://github.com/SuperKogito/Voice-based-gender-recognition)
-- [test url 21](https://github.com/SuperKogito/Voice-based-gender-recognition/issues)
-- [test url 22](https://github.com/SuperKogito/Voice-based-gender-recognition/issues/1)
+- [test url 10](https://github.com/SuperKogito/Voice-based-gender-recognition)
+- [test url 11](https://github.com/SuperKogito/Voice-based-gender-recognition/issues)
 - [test url 23](https://github.com/SuperKogito/Voice-based-gender-recognition/issues/2)
-- [test url 24](https://github.com/SuperKogito/Voice-based-gender-recognition/issues)
 
 - [broken_test url 1](https://none.html)
 - [broken_test url 2](https://github.com/SuperKogito/URLs-checker/README.md)
```

Comment:
not that I am against it but Is there a particular reason for deleting a bunch of links?

Reply:
For this particular set of tests, having the extra urls (with retry) didn’t benefit the test but just made it longer, so I removed some that were redundant. It definitely could still be improved upon!